ecs-agent/csiclient/csi_client_linux_integ_test.go

```diff
@@ -0,0 +1,91 @@
+//go:build linux && integration
+// +build linux,integration
+
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+package csiclient
+
+import (
+	"context"
+	"encoding/json"
+	"os/exec"
+	"strconv"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/aws/amazon-ecs-agent/ecs-agent/api/attachment/resource"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+const timeoutDuration = 1 * time.Second
+
+// Tests that CSIClient can fetch EBS volume metrics from EBS CSI Driver.
+//
+// This test assumes that there is only one EBS Volume attached to the host that's mounted
+// on "/" path. It also assumes that an EBS CSI Driver is running and listening on
+// /tmp/ebs-csi-driver.sock socket file.
+//
+// A typical environment for this test is an EC2 instance with a single EBS volume as its
+// root device volume.
+func TestNodeVolumeStats(t *testing.T) {
+	skipIfLsblkVerIsOld(t)
```

Comment:
nit: I love this method name.  Should it be `skipForUnsupportedLsblk(t)` and the supported lsblk version could be a const in this file so we could update?
```
lsblkRequredMajorVersion = 2
lsblkRequireMinorVersion = 2
lsblkRequiredPatchVersion = 27
```

Reply:
Sure! I made the update. I have added the `const`s in the `skipForUnsupportedLsblk` function itself as they are only used by that function. We can move them out when more functions need them.